app/resources/vector_resources/vector_resource_decorator.rb

```diff
@@ -104,4 +104,13 @@ def title
     return ["#{super.first} (#{portion_note.first})"] unless portion_note.blank?
     super
   end
+
+  # Overridden because we don't care if the object isn't visible - downloadable
+  # takes precedence for vector resources.
+  def downloadable?
+    return false unless respond_to?(:downloadable)
+    return visible? && public_readable_state? if downloadable.nil?
+
```

Comment:
@tpendragon can you explain a bit more this part?

Reply:
I made it so that if an item's private it's not downloadable. However, for vector resources there's a test that says even if it's private, if someone marked it downloadable you should be able to download it.

That line says if nobody's marked it downloadable, then it's downloadable if it's both visible and in a public readable state (which is either "complete" or "flagged" atm)

Reply:
Thank you.